lib/App.js

```diff
@@ -859,6 +874,9 @@ $ sudo systemctl restart docker
     const hasNodeModules = fs.existsSync(path.join(this.path, 'node_modules'));
     const hasPackageJSON = fs.existsSync(path.join(this.path, 'package.json'));
 
+    fse.ensureDirSync(path.join(this._homeyBuildPath, 'node_modules'));
```

Comment:
Why are these (and below) fs operations sync?

Reply:
And why must this happen here instead of below with the other fse ensureDirSync calls?

Reply:
> Why are these (and below) fs operations sync?

Ah, I see, this method is executed in parallel.